lib/BiochemistryAPI/utils.py

```diff
@@ -52,20 +53,20 @@ def dict_from_file(path, key='id', dialect='excel-tab'):
         id_dict[line[key]] = line
```

Comment:
I wonder why `id_dict` isn't just an ordinary `dict`. I thought `dict`s generally already respect and preserve insertion order. `OrderedDict` is a subclass of `dict` that offers some [additional functionality](https://docs.python.org/3/library/collections.html#collections.OrderedDict) that I haven't seen in these 3 files, but maybe it matters somewhere

Reply:
I've not used `OrderedDict` personally, but I wasn't aware that `dict` respects insertion order. The author liked his subclasses.